### PR TITLE
Modified Login system to allow login by nickname or email

### PR DESCRIPTION
**Pull request by pnarozniak**

### CODE REVIEW

**Review by Patrykoss — APPROVED** [review]
Ok

**Review by Patrykoss — APPROVED** [review]

File: Client/Components/LoginFormComponent.razor [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
     <div class="form-group">
-        <label for="email-input" class="d-block">Email Address</label>
-        <InputText id="email-input" @bind-Value="_loginDto.Email"/>
-        <ValidationMessage For="() => _loginDto.Email"/>
+        <label for="email-input" class="d-block">Nickname or Email</label>
+        <InputText id="email-input" @bind-Value="_loginDto.NicknameOrEmail" />
+        <ValidationMessage For="() => _loginDto.NicknameOrEmail" />
     </div>
     <div class="form-group">
         <label for="password-input" class="d-block">Password</label>
```
Comment (Ventan00) [inline Client/Components/LoginFormComponent.razor:18]: missing type="password"

**Review by Ventan00 — CHANGES_REQUESTED** [review]
As mentioned - missing input type

File: Client/Components/LoginFormComponent.razor (hunk 1 above)
Comment (xlegodroit) [reply Client/Components/LoginFormComponent.razor:18]: Whole component will be changed in future project task, this pull request refers only to login system

**Comment by pnarozniak** [comment]
Whole component will be changed in future project task I'm already working on. This pull request refers only to login system.

**Review by Ventan00 — APPROVED** [review]
Ok!

**Pull request merged**